queries/python/textobjects.scm

```diff
@@ -1,5 +1,5 @@
 (function_definition
-  body: (block)? @function.inner) @function.outer
+  body: (block)? @function.inner)
```

Comment:
I think the `?` can be removed since there are no other captures

Reply:
i found a way to write out the only definition we need instead of two different ones, and it seems to work, so i left the `?` in. but i will say that i don't really understand why it is needed. is it for when a function has no body? is such a thing valid? or is this to handle the situation where the user has not finished typing the function?

Reply:
Thanks, in this case I think it's fine. I don't even think the `?` is needed because the grammar looks like it [always gives function definitions a body](https://github.com/tree-sitter/tree-sitter-python/blob/0dee05ef958ba2eae88d1e65f24b33cad70d4367/grammar.js#L409C1-L423C7) . But I think it's fine as is

Reply:
There might be edge cases like python type stub files probably don't have a body etc.